.gitignore

```diff
@@ -20,4 +24,4 @@ standalonedata/
 
 worldwind.js
 
-worldwind.min.js
```

Comment:
No apology necessary. We haven't been clear about this convention, and I'm just now mentioning it.